### DISCUSSION TITLE
Obsolete feature in Python 3.3

### DISCUSSION THREAD

**Discussion by JacquesLB** [python-hydro/pyro2#99] [Q&A]
While using Pyro2 (for teaching purposes), I keep getting this message:

"The 'add_all' parameter of __init__() was deprecated in Matplotlib 3.3 and will be removed two minor releases later. If any parameter follows 'add_all', they should be passed as keyword, not positionally."

No harm up to now, but it might become an issue soon. Would it be possible to upgrade the code?

Thank you for your work. Cheers.

Jacques

**Comment by zingale** [comment]
Thanks for the report.  I am not sure that parameter is even needed.  I will look into this shortly.
